main.py

```diff
@@ -2,6 +2,26 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    drama = Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. If this is intentional, ensure consistency across your application. Otherwise, consider correcting it to 'Drama'.